_data/toc/software-update-guide.yml

```diff
@@ -123,6 +123,10 @@ pages:
 
         - label: Command-line upgrade with sample data
           url: /comp-mgr/cli/cli-rc1-samp.html
+        - label: (Optional) Command-line upgrade using script
+          url: /comp-mgr/cli/upgrade-with-script.html
+        - label: (Experimental) Command-line upgrade using plugin
```

Comment:
Should this be restricted to version 2.3?

Reply:
fixed